cl/recap/models.py

```diff
@@ -211,6 +212,53 @@ def file_contents(self):
     def print_file_contents(self):
         print(self.file_contents)
 
+class EmailProcessingQueue(AbstractDateTimeModel):
```

Comment:
Feel free to comment on this. I did a best effort setup from ProcessingQueue, but I am assuredly missing some things. Some fields are marked as null because the Serializer will force them to be there. Hence my other question above.